regression-test/suites/insert_p0/insert_group_commit_with_prepare_stmt_nereids.groovy

```diff
@@ -0,0 +1,111 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
We can add a loop in the original group commit regressions, for loop 0, we disable the Nereids; for loop 1, we enable the Nereids.